tests/fingerprints/functional_groups.py

```diff
@@ -8,7 +8,9 @@
 
 
 def test_functional_groups_bit_fingerprint(smiles_list, mols_list):
-    fg_fp = FunctionalGroupsFingerprint(n_jobs=-1)
+    fg_fp = FunctionalGroupsFingerprint(n_jobs=-1, verbose={
+        "leave": False,
```

Comment:
Isn't this the default option?

Reply:
It very much isn't - and the fact that it is not the default option is what made me do the PR in the first place.

Reply:
Removed these tests.